plugins/org.obeonetwork.dsl.uml2.design/src/org/obeonetwork/dsl/uml2/design/api/services/AbstractDiagramServices.java

```diff
@@ -878,6 +878,7 @@ public Element reconnectEdge(Element context, DEdge edgeView, EdgeTarget sourceV
 		}
 		reconnectService.setOldPointedElement(source);
 		reconnectService.setNewPointedElement(target);
+		reconnectService.setEdge(edgeView);
```

Comment:
I don't undertsand what is the purpose of this function. Review the name of the method ?